p2p/client_test.go

```diff
@@ -20,16 +21,41 @@ import (
 
 func TestClientStartup(t *testing.T) {
 	privKey, _, _ := crypto.GenerateEd25519Key(rand.Reader)
-	client, err := NewClient(config.P2PConfig{}, privKey, "TestChain", test.NewLogger(t))
 	assert := assert.New(t)
-	assert.NoError(err)
-	assert.NotNil(client)
+	testCases := []struct {
+		desc    string
+		p2pconf config.P2PConfig
+	}{
+		{"blank config", config.P2PConfig{}},
+		{"unblocking a not previously explicitly blocked peer shouldn't error", config.P2PConfig{
```

Comment:
[non-blocking nit]: "unblocking a peer that was not explicitly blocked before should not error" - alternative name suggestion for test case description that might be easier to read.